cli/flags.rs

```diff
@@ -282,6 +298,13 @@ fn clap_root<'a, 'b>() -> App<'a, 'b> {
         .possible_values(&["debug", "info"])
         .global(true),
     )
+    .arg(
+      Arg::with_name("quiet")
+        .short("q")
+        .long("quiet")
+        .help("Reduce verbosity")
```

Comment:
We need a better description (and long description) for this flag

Reply:
I'm not great at writing these, but took a stab anyway. Any suggestions are appreciated.